src/main/java/com/assignment/entities/FeedBySKU.java

```diff
@@ -0,0 +1,85 @@
+package com.assignment.entities;
+
+import java.util.Date;
+
+public class FeedBySKU {
```

Comment:
Add java doc
This class represents the <table name>

Reply:
done